PSReadLine/Render.Helper.cs

```diff
@@ -96,6 +96,7 @@ internal static int LengthInBufferCells(char c)
             if (c < 256)
             {
                 // We render ^C for Ctrl+C, so return 2 for control characters
+                // TODO: Do we care about this under a screen reader?
```

Comment:
I guess `^C` won't be read out? But it still takes up 2 cells even when screen reader is on.

Reply:
I checked, and the place where `^C` is specifically rendered as "^" plus "C" is `CancelLine` which is also used in screen reader mode (verified it shows up as a red ^C) so yes this part of `LengthInBufferCells` is still correct. TODO answered 😄